yb-voyager/src/callhome/diagnostics.go

```diff
@@ -362,6 +374,45 @@ type DataFileParameters struct {
 	NullString string `json:"NullString,omitempty"`
 }
 
+// =============================== Compare Performance ===============================
+/*
+Version History
+1.0: Initial version
+*/
+var COMPARE_PERFORMANCE_PAYLOAD_VERSION = "1.0"
+
+type ComparePerformancePhasePayload struct {
+	PayloadVersion    string        `json:"payload_version"`
+	TotalQueries      int           `json:"total_queries"`
+	MatchedQueries    int           `json:"matched_queries"`
+	SourceOnlyQueries int           `json:"source_only_queries"`
+	TargetOnlyQueries int           `json:"target_only_queries"`
+	QueryMetrics      []QueryMetric `json:"query_metrics"`
+	Error             string        `json:"error"`
+	ControlPlaneType  string        `json:"control_plane_type"`
+}
+
+// QueryMetric holds performance metrics for matched queries included for callhome
+type QueryMetric struct {
+	QueryLabel    string             `json:"query_label"` // "SELECT_JOIN", "INSERT_BULK", etc.
```

Comment:
I think the comment is not correct, the second word is COMPLEX, SIMPLE, etc.. right?